### DISCUSSION TITLE
how to add server name in user personal chat from bot like

### DISCUSSION THREAD

**Discussion by ORGINRISHITH** [Rapptz/discord.py#8097] [Q&A]
**like:- "hey you are found swearing(the server name here)"**

**Comment by jelni** [comment]
There is not enough context to properly answer your question, but if you have a [`Message`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Message) or a [`Context`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Context) object you can read it's `guild` attribute.

```py
async def some_handler(message: Message):
  guild = message.guild
  await message.author.send(f"Don't swear in {guild} ({guild.id})")
```
